run_dir/static/js/pricing_quote.js

```diff
@@ -242,6 +250,9 @@ app.component('v-pricing-quote', {
             this.$root.fetchExchangeRates(sel_data['exchange_rate_issued_date'])
             this.loaded_version_desc = 'Version: '+ loaded_version + ' \n' +
                                     'Agreement_number: '+this.proj_id+'_'+timestamp_val
+            if('template_text' in sel_data){
```

Comment:
Is there a project in dev that has a template text we can see?

Reply:
P28959